src/Core.php

```diff
@@ -3,36 +3,80 @@
 namespace UCSC\Blocks;
 
 use UCSC\Blocks\Assets\Assets_Subscriber;
+use UCSC\Blocks\Blocks\Featured_Block;
 use UCSC\Blocks\Blocks\News_Block;
 use UCSC\Blocks\Hooks\News_Blocks_Hooks;
+use UCSC\Blocks\Hooks\Taxonomies_Hooks;
 
 class Core {
+
+	public const BLOCKS_PUBLIC = [
+		News_Block::class => '/src/views/news_block',
+	];
+	
+	public const BLOCKS_NEWS_ONLY = [
+		Featured_Block::class => '/src/views/featured_block',
+	];
 	
 	public function init(): void {
 		$this->blocks();
 		$this->scripts();
 	}
 	
 	protected function blocks(): void {
-		add_action( 'init', static function (): void {
-			register_block_type( UCSC_DIR . '/src/views/news_block' );
-			( new News_Block() )->init();
+		add_action( 'init', function (): void {
+			$this->init_blocks();
 			( new News_Blocks_Hooks() )->hooks();
 			( new Assets_Subscriber() )->register();
+
+			if ( ! defined( 'UCSC_NEWS_SITE' ) && ! UCSC_NEWS_SITE ) {
```

Comment:
Doesn't this check if the `UCSC_NEWS_SITE` constant is not defined and then, if it's not, check it's value?

```suggestion
			if ( ! defined( 'UCSC_NEWS_SITE' ) || ! UCSC_NEWS_SITE ) {
```

Reply:
That will be addressed in #34 [here](https://github.com/ucsc/ucsc-custom-functionality/pull/34/files#diff-3822a790a5cac9e3e980820c9ad0d8184339090889a97e65ea3b1a81a63ab52fR116-R118)